tests/helpers.py

```diff
@@ -29,7 +38,16 @@ def step_env_with_gym_quirks(test_case,
 
     env.reset()
     for _ in range(n):
-        _, _, done, _ = env.step(env.action_space.sample())
+        a = env.action_space.sample()
+        a_copy = a if isinstance(a, int) else a.copy()
+        _, _, done, _ = env.step(a)
+
+        # Check for side effects
+        if isinstance(a, int):
+            assert a == a_copy
```

Comment:
Some error messages would be useful here, so if another guy finds this issue in Travis, it takes less time to figure out what exactly went wrong.

Reply:
done